warehouse/models/payments_views/payments_rides.sql

```diff
@@ -22,6 +22,7 @@
 ) }}",
 " {{ create_row_access_policy(
     principals = ['serviceAccount:metabase@cal-itp-data-infra.iam.gserviceaccount.com',
```

Comment:
@atvaccaro is there any way to safely add a comment here? but also - I took a look at the dbt docs and [it looks like bigquery doesn't allow you to get user from the target context](https://docs.getdbt.com/reference/dbt-jinja-functions/target#bigquery)

Reply:
I would just add a SQL comment outside of the body of the macro call